dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops

```diff
@@ -119,6 +119,16 @@ JSON_ENTRY_WRONG_JSON_TYPE=CWWKS5386E: The value for the key [{0}] in the JSON d
 JSON_ENTRY_WRONG_JSON_TYPE.explanation=The value for the specified key did not have the correct type. The JSON data might be malformed or might have an unexpected structure.
 JSON_ENTRY_WRONG_JSON_TYPE.useraction=Check the structure of the JSON data. Check the JSON data to see whether an error occurred.
 
+# 0=Exception message
+INTROSPECT_ERROR_GETTING_USER_INFO=CWWKS5387E: The response from the token introspection API cannot be processed: {0}
+INTROSPECT_ERROR_GETTING_USER_INFO.explanation=Information about the authenticated user cannot be obtained because another error occurred.
+INTROSPECT_ERROR_GETTING_USER_INFO.useraction=If the message contains another error message, see the user action for that message. Otherwise, check the server logs for more error messages that might indicate where the other error occurred.
+
+# 0=User API response string
+INTROSPECT_USER_API_ACTIVE_NOT_TRUE=CWWKS5388E: The active value from the introspect user API is false or is not a boolean. The full response is {0}.
+INTROSPECT_USER_API_ACTIVE_NOT_TRUE.explanation=The response from the Kubernetes user API is expected to be true.
+INTROSPECT_USER_API_ACTIVE_NOT_TRUE.useraction=Verify that the active value in the introspect user api is true.
```

Comment:
In this case, verifying the active value in the response isn't much of a remedy. That would be a bit like asking someone to verify that their check bounced after their check was rejected (when they should check their account balance, for instance). So for the NLS `useraction`, ask yourself what would someone who sees this message (here, that'd be a server administrator who controls the server configuration and environment) actually be able to do to fix the situation, or at least have a chance at getting more information? The most likely things that happened: the token is revoked or expired, the protected resource does not have permission to introspect the token, or an incorrect token introspection endpoint is used. So what would the server admin be able to do about those situations? Make sure the token is valid, verify that the token can be introspected, and verify that the correct token introspection endpoint is used.